### ISSUE TITLE
Publish new release to PyPI

### ISSUE THREAD

**Issue by christopher-dG** [tomv564/pyls-mypy#44]
Could we get a new release published with #38 included?

**Comment by ccordoba12** [comment]
Hey @tomv564, we're trying to add support for MyPy in Spyder through the `python-language-server` and we're running into troubles because the current release imports the `future` package in Python 3.

I noticed that a fix was already submitted and merged in March (PR #38), but there hasn't been a release yet. So please let us know if this package is unmaintained to decide if we need move it to the  `python-language-server` (preserving its history of commits, of course) and maintain it there.

**Comment by ccordoba12** [comment]
@tomv564, it seems you've been a bit off the grid during the last months on Github. So i hope you don't mind we take over maintenance of your work.

If we don't hear from you in the next hours, we'll move this plugin to the `python-language-server` and won't load it there if it's found by setuptools. Thanks a lot for your work on this!

**Comment by krassowski** [comment]
Any news on an update or fork?

**Comment by ccordoba12** [comment]
No, there aren't. We're trying to fix some maintenance issues with the PyLS before proceeding with this.
